src/OneBeyond.Studio.Infrastructure.Azure/Storage/BlobContainer.cs

```diff
@@ -21,4 +34,21 @@ public static class BlobContainer
 
         return blobContainerClient;
     }
+
+    private static async Task<BlobContainerClient> GetOrCreateManagedIdentityAsync(
```

Comment:
I would rather call this method `GetOrCreateAzureIdentityAsync`. Managed identity is just an option to use for authentication, it could be env variables, workload identity etc.

Reply:
Hm... So how we then say which auth option we're using?

Reply:
@fabiomaistro, you mean in the future we can introduce more parameters and a big if to decide which auth method to use.

Reply:
Changed code to be a bit cleaner.

Reply:
@andriikaplanovskyi @fabiomaistro It's actually handled automatically, we don't need to provide more info on which auth method to use, `DefaultAzureCredentials` does that automatically in the following sequence:
![image](https://github.com/onebeyond/onebeyond-studio-core/assets/103195336/7fc7f1ce-76d0-4bab-a64b-d993404a6a37)

So let's say the app is running in an AKS pod with AKS workload identity enabled it will use workload identity (a sidecar container injects the necessary info...), or if the code is running in an Azure function with a managed identity attached to it, then it will use that to authenticate etc.